pkg/cmd/server/origin/master.go

```diff
@@ -517,6 +550,13 @@ func forbidden(reason string, w http.ResponseWriter, req *http.Request) {
 	fmt.Fprintf(w, "Forbidden: %q %s", req.RequestURI, reason)
 }
 
+// RunProjectAuthorizationCache starts the project authorization cache
+func (c *MasterConfig) RunProjectAuthorizationCache() {
+	// TODO: look at exposing a configuration option in future to control how often we run this loop
+	period, _ := time.ParseDuration("1s")
```

Comment:
and once we get a high-water mark in a store, every duration period if there was no change will just say, was there a change?  no, return.  anyway, this to me is how good of a first impression do you want in your beta where you create the project, and then want to list it and see it there.